cli/deploy.ts

```diff
@@ -132,15 +135,22 @@ const createSchemas = async (schemaNames: string[]) => {
           } else if (status.isInBlock || status.isFinalized) {
             const evt = eventWithSectionAndMethod(events, "schemas", "SchemaCreated");
             if (evt) {
-              const val = evt?.data[1];
-              console.log("SUCCESS: " + schemaName + " schema created with id of " + val);
-              resolve({ schemaName, id: Number(val.toHuman()) });
-            } else resolve({ schemaName });
+              const id = evt?.data[1];
+              console.log("SUCCESS: " + schemaName + " schema created with id of " + id);
+              const v2n = Object.fromEntries([[schemaDeploy.dsnpVersion, Number(id.toHuman())]]);
+              resolve([schemaName as DsnpSchemaName, v2n]);
+            } else {
+              console.error("ERROR: SchemaCreated event not found");
+              reject();
```

Comment:
nit: maybe reject with the error string?
```suggestion
              reject("SchemaCreated event not found");
```

Reply:
Thanks, added messages to all the `reject()` calls. (I'm getting better at handling rejection...)